Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/RiskLevelRepository.kt

```diff
@@ -7,14 +7,28 @@ import kotlinx.coroutines.flow.MutableStateFlow
 
 object RiskLevelRepository {
 
-    private val internalRisklevelScore = MutableStateFlow(getLastSuccessfullyCalculatedScore().raw)
+    private val internalRisklevelScore = setInternalRiskLevelScore()
     val riskLevelScore: Flow<Int> = internalRisklevelScore
 
     private val internalRiskLevelScoreLastSuccessfulCalculated =
         MutableStateFlow(LocalData.lastSuccessfullyCalculatedRiskLevel().raw)
     val riskLevelScoreLastSuccessfulCalculated: Flow<Int> =
         internalRiskLevelScoreLastSuccessfulCalculated
 
+    /**
+     * Set the internal risk level score [internalRisklevelScore]
+     */
+    private fun setInternalRiskLevelScore(): MutableStateFlow<Int> {
+        val lastSuccessfullyCalculatedScore = getLastCalculatedScore().raw
```

Comment:
Thanks! I still don't get though, why we do this here in the Repo and not at the source in  LocalData.lastCalculatedRiskLevel().
Currently, we have overwriting behavior in multiple places.

Reply:
No, I think you're correct and I have adjusted the code to match this.

Reply:
@ralfgehrer @InterestedParties 
what do you think about the UNDETERMINED state now?

Reply:
@BMItter not quite sure I understand your question.

This is what it says in our code. Hence, initializing sth with that value is probably not the best idea.
`  
    // mapped to no UI state
    // this should never happen
    UNDETERMINED(RiskLevelConstants.UNDETERMINED);
`